app/MMB/src/navigation/DetailShareButton.js

```diff
@@ -14,7 +14,7 @@ type PropsWithContext = {|
 
 class DetailShareButton extends React.Component<PropsWithContext> {
   onPress = () => {
-    const lang = GetDeviceLocale().split('-')[0] || 'en';
+    const lang = GetDeviceLocale.getLanguage() || 'en';
```

Comment:
Before this file was just about DeviceLocale, but now it is about more. Maybe it would be better to rename it to DeviceInfo

`DeviceInfo.getLanguage()` makes more sense than `GetDeviceLocale.getLanguage()`

Reply:
Agree, now when functions are called `get`, the prefix in `DeviceInfo` is unnecessary.